src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -95,7 +95,7 @@ PlusStatus vtkPlusWinProbeVideoSource::ReadConfiguration(vtkXMLDataElement* root
       int mwidthSeconds = std::stoi(mwidthSeconds_string);
       if(mwidthSeconds > 0)
       {
-        m_MWidth = this->MWidthFromSeconds(mwidthSeconds);
+        m_ExtraFrameSize[1] = this->MWidthFromSeconds(mwidthSeconds); // [0] or [1]?
```

Comment:
` // [0] or [1]?` as it related to this comment, I'm assuming this was figured out?

Reply:
@dzenanz Do you know about this?

Reply:
Since this is the first change in the file, I might have put the comment there. I think I tested this, but testing again with M-mode would not hurt. The comment should be removed.

Reply:
I get a bunch of these errors when going from "M" mode to "BRF" mode which might be related to here? `|ERROR|115.700000| vtkPlusBuffer: Unable to add frame to video buffer - frame format doesn't match!`

Reply:
The warning directly before the error:
`|WARNING|028.599000| VideoDeviceLinearArray-ExtraSourceVideo: Frame format and buffer frame format does not match (expected frame size: 128x512x1  received: 1536x128x1)!| in C:\SP\Adam\SPR15_x64\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(198)`

Reply:
I've pushed a fix for this, there was a spot where the sizes passed to `adjustBufferSizes` were incorrect for brf mode.